File: elixir/equalwords_short.exs
File.read!("words.txt")|>String.split("\n")|>Enum.reduce(%{},fn(a, b)->c=String.split(a,"")|>Enum.sort|>List.to_string
d=Map.get(b,c,"")
Map.put(b,c,"#{d}, #{a}")end)|>Map.values|>Enum.each(fn(a)->String.split_at(a,2)|>elem(1)|>IO.puts
end)

File: go/equalwordsShort.go
package main
import(e "sort"
f "fmt"
g "io/ioutil"
h "strings"
)
func main(){c,_:=g.ReadFile("words.txt");var a=make(map[string][]string);for _,i:=range h.Split(string(c),"\n"){var j=h.Split(i,"");e.Strings(j);var k=h.Join(j,"");a[k]=append(a[k],i)};for _,v:=range a{f.Println(h.Join(v,", "))}}